src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -397,12 +589,60 @@
             "MyExtensionsApp.Skia.Linux.FrameBuffer/**/*"
           ]
         },
+        {
+          "condition": "(winAppSdk && skia-gtk)",
+          "exclude": [
+            "MyExtensionsApp.Skia.Gtk/Package.appxmanifest",
```

Comment:
Shouldn't this just exclude the whole skia.gtk folder?

Reply:
no the condition excludes duplicated files that are referenced from the WinAppSdk project but only when you include WinAppSdk. If you exclude WinAppSdk but have Gtk & Linux FB they will duplicate the files rather than using the same version between projects.